fix: adjust details about padding / cutting rules + add summary table in LSP2 + LSP8 (#262)
lukso-network/LIPs: LSPs/LSP-2-ERC725YJSONSchema.md

`<mixed type>` can be one of `uint<M>`, `address`, `bool` or `bytes<M>` types.

**The following padding and cutting rules apply:**

#### Left padded types

- `bool` will be left padded and left-cut

> - `uint<M>` will be left padded and **left-cut if `M` is larger `160` bits** (= the 20 right most bytes are kept).

Reviewer: ```suggestion
- `uint<M>` will be left padded and **left-cut if `M` is larger than `160` bits** (= the 20 rightmost bytes are kept).
```

#### Right padded types

> - `bytes<N>` will be right padded and **right cut if `N` is larger `20 bytes`** (= the 20 left most bytes are kept).

Reviewer: ```suggestion
- `bytes<N>` will be right padded and **right cut if `N` is larger than `20 bytes`** (= the 20 leftmost bytes are kept).
```
